core/tauri/src/window/mod.rs

```diff
@@ -935,6 +935,13 @@ impl<'a, R: Runtime> WindowBuilder<'a, R> {
     self.webview_attributes.incognito = incognito;
     self
   }
+
+  /// Enable proxy for the WebView
+  #[must_use]
+  pub fn proxy_config(mut self, proxy: &str) -> Self {
```

Comment:
Let's rename this and accept a `Url` instead of `&str`
```suggestion
  pub fn proxy_url(mut self, url: Url) -> Self {
```